anabot/runtime/installation/hub/connect_to_redhat/__init__.py

```diff
@@ -309,7 +336,7 @@ def unregister_button(local_node, visible=True, sensitive=True):
 def wait_until_registered_handler(element, app_node, local_node):
     register_button = getnode(local_node, "push button", tr("_Register", context="GUI|Subscription|Register"), sensitive=None, visible=None)
     status_label = getsibling(register_button, -1, "label", sensitive=None)
-    tr_register_phases = [ tr(phase) for phase in register_phases ]
+    tr_register_phases = [tr(phase) for phase in register_phases]
```

Comment:
Another formatting change that should be either removed or moved to separate commit.